Things.md

```diff
@@ -114,4 +116,19 @@ FSM.setOpacity(thing, 1);
 
 // Sets a Thing's opacity to 0 (transparent). 
 FSM.setOpacity(thing, 0);
-```
\ No newline at end of file
+```
+
+# Adding Things  
+You're able to add Things to the game via `addThing`, specifying the `x` and `y` positions (again this is relative to the top left corner of the screen). The `x` is where the Thing's `left` bound will be and the `y` is where the Thing's `top` bound will be. You can either add it using an actual Thing object or the class name for the Thing:
+```javascript
+// Adds a Thing to the game at position (1,2) using a Thing object.
+FSM.addThing(thing, 1, 2);
+
+// Adds a Thing to the game at position (1,2) using the class name.
+FSM.addThing("CastleFireball", 1, 2);
+
+// Adds a Thing next to (to the right of) another Thing.
+FSM.addThing(thing, otherThing.right, otherThing.top);
```

Comment:
I like this example. It would be nice to have one for above another thing also, as it would show the slightly more complicated math.